src/PointwiseFunctions/SpecialRelativity/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY SpecialRelativity)
```

Comment:
Since @fmahebert has been refactoring how these libraries are set up, would you mind updating this CMake file? You can use `DataStructures/CMakeListst.txt` as an example.. we are going to also track header files now